<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>liveGUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc/icons.qrc">
    <normaloff>:/icons/icons/livestream-ICON.png</normaloff>:/icons/icons/livestream-ICON.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="topLayout">
      <item>
       <layout class="QHBoxLayout" name="buttonLayout_2">
        <item>
         <layout class="QVBoxLayout" name="adressOuterLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Stream adress</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="adressInnerLayout">
            <item>
             <widget class="QLineEdit" name="adressEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="qualityLayout">
          <item>
           <widget class="QLabel" name="qualityLabel">
            <property name="text">
             <string>quality:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="qualityComboBox">
            <property name="currentText">
             <string>best</string>
            </property>
            <item>
             <property name="text">
              <string>best</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>high</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>medium</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>low</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>worst</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="playButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../qrc/icons.qrc">
            <normaloff>:/icons/icons/play.png</normaloff>:/icons/icons/play.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../qrc/icons.qrc">
            <normaloff>:/icons/icons/stop.png</normaloff>:/icons/icons/stop.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="browseTab">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="icon">
        <iconset resource="../qrc/icons.qrc">
         <normaloff>:/icons/icons/remote-control.png</normaloff>:/icons/icons/remote-control.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Browse</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="BrowsingWidget" name="browsingWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="favoritesTab">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="icon">
        <iconset resource="../qrc/icons.qrc">
         <normaloff>:/icons/icons/browse-favs.png</normaloff>:/icons/icons/browse-favs.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Favorites</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="FavoriteWidget" name="favoriteWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="playerInfo">
       <property name="accessibleName">
        <string/>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="icon">
        <iconset resource="../qrc/icons.qrc">
         <normaloff>:/icons/icons/console.png</normaloff>:/icons/icons/console.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Output</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="liveStreamLabel">
          <property name="text">
           <string>Livestreamer output:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="liveStreamOutput">
          <property name="midLineWidth">
           <number>1</number>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="previewTab">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="icon">
        <iconset resource="../qrc/icons.qrc">
         <normaloff>:/icons/icons/preview.png</normaloff>:/icons/icons/preview.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Preview</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QScrollArea" name="previewScroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>506</width>
             <height>333</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="PreviewStreamWidget" name="previewStreamWidget" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArkiv">
    <property name="title">
     <string>Arkiv</string>
    </property>
    <addaction name="actionAvsluta"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionNetwork"/>
   </widget>
   <addaction name="menuArkiv"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAvsluta">
   <property name="text">
    <string>Avsluta</string>
   </property>
  </action>
  <action name="actionNetwork">
   <property name="text">
    <string>Network</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FavoriteWidget</class>
   <extends>QWidget</extends>
   <header>favoritewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PreviewStreamWidget</class>
   <extends>QWidget</extends>
   <header>previewstreamwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BrowsingWidget</class>
   <extends>QWidget</extends>
   <header>browsingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qrc/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionAvsluta</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>updateLivestreamOutput(QString)</signal>
   <receiver>liveStreamOutput</receiver>
   <slot>appendPlainText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>updateLivestreamOutput(QString)</signal>
 </slots>
</ui>
